src/Event/AuditableSubscriber.php

```diff
@@ -148,6 +194,14 @@ protected function createLogEntry(object $entity): void
                 $accessor->setValue($entity, $config['commentProperty'], null);
             }
 
+            if ($this->store) {
+                $entityHash  = spl_object_hash($entity);
```

Comment:
Is this line is ok? Because `spl_object_hash()` can reuse previous values after cleaning unused objects. For example, see https://3v4l.org/oU6uM